ckanext/xloader/utils.py

```diff
@@ -210,5 +214,7 @@ def type_guess(rows, types=TYPES, strict=False):
         # element in case of a tie
         # See: http://stackoverflow.com/a/6783101/214950
         guesses_tuples = [(t, guess[t]) for t in types if t in guess]
+        if not guesses_tuples:
```

Comment:
How does this handle files with only a header and no data rows?

Reply:
@ThrawnCA can you add a test for a header only upload please.

Reply:
@duttonw I can't; the pull request doesn't originate from our repo.